### DISCUSSION TITLE
Does upscayl set an invisible watermark anywhere during the process?

### DISCUSSION THREAD

**Discussion by Kornifikator** [upscayl/upscayl#799] [Q&A]
Hello, my question is simple:
Does upscayl set an invisible watermark anywhere during the entire process, like most diffusers do, to mark images as AI-generated/-enhanced/-processed/-manipulated/etc.?

Thank you in advance

**Comment by NayamAmarshe** [comment]
From the description: 
> 🍿 Upscayl is a free and open source desktop application that lets you upscale your low resolution images using advanced AI Models.

What do you think? :grin: Try it!

**Comment by aaronliu0130** [comment]
No. But don't claim you made stuff on your own, please 😌
